client/tests/e2e/helpers/utils.ts

```diff
@@ -328,11 +328,11 @@ export async function login(homePage: Page, name: string): Promise<void> {
 
 export async function logout(page: MsPage): Promise<void> {
   await page.locator('.topbar').locator('.profile-header').click();
-  const buttons = page.locator('.profile-header-popover').locator('.main-list').getByRole('listitem');
+  const buttons = page.locator('.profile-header-organization-popover').locator('.main-list').getByRole('listitem');
   await buttons.nth(4).click();
   await answerQuestion(page, true);
   await expect(page).toBeHomePage();
-  await expect(page.locator('.homepage-header').locator('.topbar-left')).toHaveText('Welcome to Parsec');
+  await expect(page.locator('.homepage-header').locator('.topbar-left')).toHaveText('Welcome to ParsecAccess your organizations');
```

Comment:
Split that in two to check the element with `Welcome to Parsec` amd the element with `Access your organizations`